packages/framework/assets/js/admin/validation/form/mailTemplate.js

```diff
@@ -20,4 +20,4 @@ export default function validationMailTemplate () {
     });
 }
 
-(new Register()).registerCallback(validationMailTemplate);
+(new Register()).registerCallback(validationMailTemplate, 'mailTemplate.validationMailTemplate');
```

Comment:
is `mailTemplate.validationMailTemplate` correct? For example `validationAdvert` uses just `validationAdvert` without any prefix

Reply:
1c9bd3f272d46c82caa4dc1e1722134ef74d13f8